mozilla/bedrock
# [fix bug 1409779] Update Firefox sub navigation for Quantum

## Description
- Updates Firefox sub navigation and global nav menu links for Fireofx 57.
- New links are behind the switch `firefox-57-release`.
- Note: there's one new string here `Extensions`, all the others (`Desktop`, `Mobile`, `Blog`) are already in [main.lang](https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/main.lang). 
- For non-English locales, I've made `Extensions` fallback to `Add-ons`. We would still like to translate `Extensions` though, as it's the new term we should use moving forward.
- Note: this PR should not merge until after #5224 so I can update the links to `/firefox/mobile/`.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1409779

## Testing
Demo: https://www-demo3.allizom.org/en-US/firefox/
- Make sure the new links appear as expected when the switch is enabled, and old content shows when disabled.
- Check for typos?
